File: 1470-shuffle-the-array/1470-shuffle-the-array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> a;
        if ((nums.size())%2!=0){
            return nums;
        }
        else{
            for (int i=0; i<(nums.size()/2); i++){
                a.emplace_back(nums[i]);
                a.emplace_back(nums[(nums.size()/2)+i]);
            }
        }
        return a;
    }
};

File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        a= []
        b= len(nums)
        if (b%2==1):
            return nums
        elif (b%2==0):
            for i in range(b//2):
                a.append(nums[i])
                a.append(nums[(b//2)+i])
            return a